src/newgrf.cpp

```diff
@@ -6630,10 +6632,6 @@ static void SkipIf(ByteReader *buf)
 				break;
 			case 0x10: result = GetRoadTypeByLabel(BSWAP32(cond_val)) != INVALID_ROADTYPE;
 				break;
-			case 0x11: result = GetRoadTypeByLabel(BSWAP32(cond_val)) == INVALID_ROADTYPE;
-				break;
```

Comment:
I think this stuff is correct and should be kept.
There are also separate translation tables for road and tram.

But who knows.

Reply:
If they should be kept as a separation between Road and Tram, they should probably be extended with actually different checks.